.github/workflows/trigger-react.yml

```diff
@@ -0,0 +1,25 @@
+name: Trigger REACT Buildkite Pipeline
+
+on:
+  push:
+    branches: [ "main", "7.*", "8.*" ]
+
+jobs:
+  trigger_buildkite:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger a Buildkite REACT Build
+        run: |
+          curl -X POST "https://api.buildkite.com/v2/organizations/elastic/pipelines/react/builds" \
+            -H "Authorization: Bearer ${{ secrets.REACT_BUILDKITE_API_TOKEN }}" \
+            -H "Content-Type: application/json" \
+            -d '{
+              "commit": "HEAD",
+              "branch": "main",
```

Comment:
will this always be main? What does this specify?

Reply:
Yes. this means run on the main branch of `react`.